onnxruntime/core/optimizer/transpose_optimizer/transpose_optimizer.cc

```diff
@@ -1756,6 +1756,33 @@ static const HandlerInfo* GetHandler(api::NodeRef& node, bool allow_extended_ops
   return nullptr;
 }
 
+// Some op should be optimized any time there is a transpose as input and a handler is available.
+static bool isOpSkipCostCheck(const OptimizerCtx& ctx, const api::NodeRef& node) {
+  if (node.IsOp("Transpose")) {
+    return true;
+  }
+  if (node.IsOp("MaxPool")) {
+    // Inclusion of MaxPool is a hack because it has higher perf in the NHWC variant when supported.
+    return true;
+  }
+#if defined(_M_ARM64) || defined(__aarch64__) || defined(_M_ARM) || defined(__arm__)
+  if (node.IsOp("Resize")) {
+    // Inclusion of Resize is a hack because it has higher perf in the NHWC variant when
```

Comment:
>hack [](http://example.com/codeflow?start=32&length=4)

this is not hack. It is an special optmization.

Reply:
Fixed